app/controllers/sellers_controller.rb

```diff
@@ -2,6 +2,7 @@ class SellersController < ApplicationController
   def new
     @seller = Seller.new
   end
+
   def create
     if current_user.role.eql?('seller') 
       if current_user.seller.present?
```

Comment:
@seller = Seller.new(seller_params)
@seller.save
Why are you using  instance variable ? If there is no need to write instance variable . you can use local variable

Reply:
sure

Reply:
lot of factors depend upon the new and create action, during the new record is generated we have to use @instance variable in new and create action. it's also being accessible for view here is the reference https://guides.rubyonrails.org/getting_started.html